app/models/reports/instrument_access_list_export_raw.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "csv"
+
+module Reports
+
+  class InstrumentAccessListExportRaw
```

Comment:
Instruments aren't the only product that can have an access list, and Export Raw is an existing feature.  How about:
```suggestion
  class ProductAccessListCsvReport
```